app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -279,7 +279,8 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
     isNegativeDepositAmount
 
   const showSetAllowance =
-    allowanceFinished || hasZeroAllowance === Ternary.True || hasEnoughAllowance === Ternary.False
+    !cpk?.cpk.isSafeApp() &&
```

Comment:
Do you not need to set allowance in the safe app?

Reply:
No the allowance is usually to allow the cpk to transfer user funds. The safe is the user here and is using its own funds.